gapir/cc/cached_resource_loader.cpp

```diff
@@ -57,15 +59,14 @@ bool CachedResourceLoader::load(const Resource* resources, size_t count,
     return false;  // Not enough space
   }
 
-  bool succeeded = true;
-
   ResourceLoadingBatch batch;
   uint8_t* dst = reinterpret_cast<uint8_t*>(target);
   for (size_t i = 0; i < count; i++) {
     const auto& r = resources[i];
     // Check cache first
     if (mCache->hasCache(r)) {
-      succeeded &= mCache->loadCache(r, static_cast<void*>(dst));
```

Comment:
There was no check on "succeeded" at the end? Strange.

Reply:
This was because 
`
if (mCache->hasCache(r)) == true implies   mCache->loadCache(r, static_cast<void*>(dst)); == true
`

And we have no other option if we cant actually load from the cache anyway (if it is in there), so asserting is likely the way to go.